paper/paper.md

```diff
@@ -0,0 +1,99 @@
+---
+title: 'opt_einsum: A Python package for optimizing contraction order for einsum-like expressions`
+tags:
+  - array
+  - tensors
+  - optimization
+  - phylogenetics
+  - natural selection
+  - molecular evolution
+authors:
+ - name: Daniel G. A. Smith
+   orcid: 0000-0001-8626-0900
+   affiliation: "1"
+ - name: Johnnie Gray
+   orcid: 0000-000x-xxxx-xxxx
```

Comment:
My orcid is 0000-0001-9461-3024
And my affiliation is "University College London"